src/Query/TaskGroup.ts

```diff
@@ -1,25 +1,31 @@
 import type { Task } from '../Task';
-import type { GroupHeading } from './GroupHeading';
+import type { GroupDisplayHeading } from './GroupDisplayHeading';
 
 /**
- * TaskGroup stores a single group of tasks, that all share the same group names.
- * TaskGroup objects are stored in a TaskGroups object.
+ * Store a single group of tasks, that all share the same group names.
```

Comment:
`tasks, that all share the same group names` - you mean same `GroupDisplayHeading`? If so, why `TaskGroup.groupHeadings` is an Array and not a single object?